init.lua

```diff
@@ -43,13 +51,15 @@ highlight_mouse_cursor.bind_hotkey(KBD_WIN_MOVE, "m")
 --[[ HYPER-LAUNCH ]]
 
 local launch = require("launch")
-hyper_or_esc.bind("f", launch.new_finder_window)
-hyper_or_esc.bind("b", launch.new_chrome_window)
-hyper_or_esc.bind("t", launch.new_iterm2_window)
-hyper_or_esc.bind("k", launch.launch_mac_pass)
-hyper_or_esc.bind("n", launch.launch_notes)
+hyper_or_esc.bind("space", function() hs.application.launchOrFocus("Google Chrome") end)
```

Comment:
... so these changes make me realize that what the project *should* have is an `examples/` directory, and in it an `init.lua.example` file which the user should copy and modify. I will do this, but in the mean time can you remove `init.lua` from the PR please?